examples/subplots_axes_and_figures/ganged_plots.py

```diff
@@ -0,0 +1,40 @@
+"""
+==========================
+Creating adjacent subplots
+==========================
+
+To create plots that share a common axes (visually) you can set the
+hspace between the subplots close to zero (do not use zero itself). Passing
```

Comment:
Using zero seems to work, perhaps it was an old limitation.  Please check, and remove this note and use 0 below instead of 0.001, if you agree.